src/components/ActiveFiltersQuery/ActiveFiltersQuery.js

```diff
@@ -13,8 +13,9 @@ const ActiveFiltersQuery = ({ classes }) => {
 
   // activeFilters helps filtering only active Filters
   const activeFilters = [];
-
-  // Loop all filters to find active ones only
+  // Loop all filters except age to find active ones only
+  const ageData = allFiltersinfo.data.filter((sideBar) => sideBar.slider === true);
```

Comment:
As you are cleaning can you rename the variable name

Reply:
The variable is renamed.